celery/bin/shell.py

```diff
@@ -117,7 +119,7 @@ def _invoke_default_shell(locals):
 @handle_preload_options
 def shell(ctx, ipython=False, bpython=False,
           python=False, without_tasks=False, eventlet=False,
-          gevent=False):
+          gevent=False, **kwargs):
```

Comment:
The **kwargs aspect of this seems to be also required and related to https://github.com/celery/celery/pull/8049